Classes/Emogrifier.php

```diff
@@ -1303,9 +1303,9 @@ private function matchIdAttributes(array $match)
      */
     private function matchClassAttributes(array $match)
     {
-        return ($match[1] !== '' ? $match[1] : '*') . '[contains(concat(" ",@class," "),concat(" ","' .
+        return ($match[1] !== '' ? $match[1] : '*') . '[contains(concat(" ",normalize-space(@class)," "),concat(" ","' .
             implode(
-                '"," "))][contains(concat(" ",@class," "),concat(" ","',
+                '"," "))][contains(concat(" ",normalize-space(@class)," "),concat(" ","',
                 explode('.', substr($match[2], 1))
             ) . '"," "))]';
```

Comment:
Are currently both changes covered by the unit test, i.e., will the test still fail if I remove either of the changes?